### ISSUE TITLE
joycon bug but i have no clue where to post bugs.

### ISSUE THREAD

**Issue by Yolotroll101** [lakka-switch/documentation#7]
I found a bug with the joycon moving mario foward in sm64. This only happens with it, and no other game. When playing on the switches OS, doesn't happen there either. I've tried re-calibrating the joycon on the switch OS, but the bug still persists in mario 64. I _do_ have another joycon, and it works fine with it, but it's kinda beaten up on the outside so i'd prefer to not use it, and I cannot buy another seeing as I am pretty much broke.

Basically, I cannot play SM64 with this joycon bug happening.
